src/js/utils/log.js

```diff
@@ -8,9 +8,13 @@ import {isObject} from './obj';
 
 let log;
 
+// This is the private tracking variable for logging level.
+let level = 'all';
```

Comment:
It may be better to set the default log level in the same location  as the check for `window.VIDEOJS_DEFAULT_LOG_LEVEL`